runtime/tests/jniarg/cdefs.c

```diff
@@ -153,5 +153,11 @@ void JNICALL Java_JniArgTests_summary( JNIEnv *p_env, jobject p_this )
 	} else {
 		j9tty_printf( PORTLIB, "JNI ARG Tests PASSED\n" );
 	}
+
+	if (argFailures + retValFailures > 0) {
```

Comment:
I would add brackets here.

Reply:
Fixed in 6d9d6bd (see [force push](https://github.com/eclipse/openj9/compare/948360bc778a1a5476d4a220127a3e9a0c4678ac..6d9d6bd876f3bef3b05bf6ab8eed33d3359c97e2))